l10n_it_delivery_note/models/account_invoice.py

```diff
@@ -157,9 +154,10 @@ def update_delivery_note_lines(self):
                                 self._prepare_note_dn_value(sequence, dn),
                             )
                         )
-                    for invoice_line in dn_invoice_lines:
                         sequence += 1
+                    for invoice_line in dn_invoice_lines:
                         invoice_line.sequence = sequence
+                        sequence += 1
```

Comment:
**question:** a livello funzionale non dovrebbe cambiare nulla, ma come mai incrementi di 1 la sequenza anche prima di entrare nel ciclo for?